audbackend/core/backend.py

```diff
@@ -263,19 +266,23 @@ def get_file(
 
         audeer.mkdir(dst_root)
         if (
-                not os.access(dst_root, os.W_OK) or
-                (os.path.exists(dst_path) and not os.access(dst_root, os.W_OK))
+            not os.access(dst_root, os.W_OK) or
+            (os.path.exists(dst_path) and not os.access(dst_root, os.W_OK))
```

Comment:
Why do we need the second line? If this is true, the first line should always be true as well.

Reply:
Thanks for spotting, the second line should check if we have permission to `dst_path`, so I fixed it to:

```
(os.path.exists(dst_path) and not os.access(dst_path, os.W_OK))
```

I also added a test for it now.